05/MenuItem.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class MenuItem extends React.Component {
+	render() {
+		const { text, url } = this.props;
+		return (
+			<li>
+				<a href={url}> {text}</a>
+			</li>
+		);
+	}
+}
+MenuItem.propTypes = {
+	text: PropTypes.string.isRequired,
+	url: PropTypes.string.isRequired,
+};
+export default MenuItem;
```

Comment:
To już 3 komponent o tej samej nazwie w tym repo :P

Reply:
czyli moglem jednak importowac z ../03/MenuItem